### PR TITLE
CMR-6079: add valid KMS values to CMR Search API docs

### PR DESCRIPTION
**Pull request by mschmele**

### CODE REVIEW

File: search-app/docs/api.md [hunk 1]
```diff
@@ -174,6 +174,16 @@ Join the [CMR Client Developer Forum](https://wiki.earthdata.nasa.gov/display/CM
...
+  * [Valid KMS Keyword Values](#valid-kms-keyword-values)
+    * [Science Keywords](#kms-science-keywords)
+    * [Platforms](#kms-platforms)
+    * [Instruments](#kms-instruments)
+    * [Projects](#kms-projects)
+    * [Providers](#kms-providers)
+    * [Spatial Keywords](#kms-spatial-keywords)
+    * [Concepts](#kms-concepts)
+    * [Iso Topic Categories](#kms-iso-topic-categories)
+    * [Granule Data Format](#kms-granule-data-format)
```
Comment (CayvonH) [inline search-app/docs/api.md]: Not sure if there is a different ticket for this but maybe add a reference to the work Leo did with measurement_names.

Comment (ygliuvt) [inline search-app/docs/api.md]: We need to add the Variable Measurement Names to the list.

Comment (mschmele) [reply search-app/docs/api.md]: Done

Comment (mschmele) [reply search-app/docs/api.md]: Done

**Pull request merged**